test/kubemark/run-e2e-tests.sh

```diff
@@ -33,10 +33,10 @@ export KUBE_MASTER_URL="https://${KUBE_MASTER_IP}"
 export KUBECONFIG="${ABSOLUTE_ROOT}/test/kubemark/kubeconfig.loc"
 export E2E_MIN_STARTUP_PODS=0
 
-if [ -z "$@" ]; then
+if [[ -z "$@" ]]; then
 	ARGS='--ginkgo.focus=should\sallow\sstarting\s30\spods\sper\snode'
 else
 	ARGS=$@
 fi
 
-${KUBE_ROOT}/hack/ginkgo-e2e.sh "--e2e-verify-service-account=false" "$ARGS"
+${KUBE_ROOT}/hack/ginkgo-e2e.sh "--e2e-verify-service-account=false" $ARGS
```

Comment:
Otherwise all ARGS will be passed as a single argument with whitespaces, not a "list" of them